### PR TITLE
Simplify benefits ahead of tier recalibration

### PR DESCRIPTION
**Pull request by davidrapson**
This PR simplifies how we describe benefits ahead of the tier recalibration launch. We can release these changes ahead of time as we're not including the new benefits here.

## Supporter pages

Before/After
screen shot 2015-08-20 at 17 46 41
screen shot 2015-08-20 at 17 40 33

## Choose tier

Before/After
choose-tier-before
after

## Change tier

Before/After
change-tier-before
screen shot 2015-08-20 at 17 44 14

## Cancel

Before/After
cancel-before
cancel-after

@afiore

### CODE REVIEW

File: frontend/app/model/Benefits.scala [hunk 1]
```diff
@@ -58,57 +55,64 @@ object Benefits {
 
   val friendBenefitsList = benefitsFilter(
     "book_tickets",
-    "digital_digest",
-    "video_highlights"
+    "offers_competitions",
```
Comment (tudorraul) [inline frontend/app/model/Benefits.scala]: Relying on strings might cause issues when typos occur. On the other hand this is contained in the same file, and the risks are small.

**Comment by tudorraul** [comment]
:+1: bar the comment

File: frontend/app/model/Benefits.scala (hunk 1 above)
Comment (davidrapson) [inline frontend/app/model/Benefits.scala]: @tudorraul I'd definitely prefer not to use strings, how would you approach this?

File: frontend/app/model/Benefits.scala [hunk 2]
```diff
@@ -24,17 +24,14 @@ object Benefits {
   )
 
   val allBenefits = Seq(
```
Comment (tudorraul) [inline frontend/app/model/Benefits.scala]: This looks to be used only in the `views` package and would be better located there, I think.

File: frontend/app/model/Benefits.scala (hunk 1 above)
Comment (tudorraul) [inline frontend/app/model/Benefits.scala]: name each `BenefitItem` object and use its name onwards

**Pull request merged**